policy/modules/contrib/fdo.te

```diff
@@ -0,0 +1,71 @@
+policy_module(fdo, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type fdo_t;
+type fdo_exec_t;
+init_daemon_domain(fdo_t, fdo_exec_t)
+
+type fdo_conf_t;
+files_config_file(fdo_conf_t)
+
+type fdo_conf_rw_t;
+files_config_file(fdo_conf_rw_t)
+
+type fdo_tmp_t;
+files_tmp_file(fdo_tmp_t)
+
+type fdo_unit_file_t;
+systemd_unit_file(fdo_unit_file_t)
+
+permissive fdo_t;
+
+########################################
+#
+# fdo local policy
+#
+allow fdo_t self:fifo_file rw_fifo_file_perms;
+allow fdo_t self:netlink_route_socket r_netlink_socket_perms;
+allow fdo_t self:tcp_socket create_stream_socket_perms;
+allow fdo_t self:udp_socket create_socket_perms;
+allow fdo_t self:unix_stream_socket create_stream_socket_perms;
+
+can_exec(fdo_t, fdo_exec_t)
+
+manage_dirs_pattern(fdo_t, fdo_conf_t, fdo_conf_t)
```

Comment:
Since there is fdo_conf_rw_t, does fdo_t still need to manage fdo_conf_t?

Reply:
Fido is creating /etc/fdo/aio dir and subdirs are either fdo_conf_t or fdo_conf_rw_t